css/style.css

```diff
@@ -0,0 +1,71 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    border: 0;
+}
+
+html {
+    font-family: 'Sen', sans-serif;
+}
+
+:root {
+    font-size: 62.5%;
+    --text-input-field: #f9f9f9;
+    --black: #000000;
+    --orange: rgb(238, 71, 5);
+    --sm: 1em;
+    --md: 1.6em;
+    --lg: 2.6em;
```

Comment:
what do these values mean? I know they are for different viewports but I don't know if they are used for box model properties, positioning, font size...